dts/riscv/egis/egis_et171.dtsi

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2025 Egis Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <mem.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		CPU0: cpu@0 {
+			compatible = "andestech,andescore-v5", "riscv";
+			device_type = "cpu";
+			reg = <0>;
+			status = "okay";
+			riscv,isa = "rv32gc_xandes";
+			mmu-type = "riscv,sv32";
+			// TODO change to config value?
+			clock-frequency = <66000000>;
+			i-cache-line-size = <32>;
+			d-cache-line-size = <32>;
+			CPU0_intc: interrupt-controller {
```

Comment:
newline on line 29 and follow dts naming requirements https://docs.zephyrproject.org/latest/contribute/style/devicetree.html also note the linux ones